public/js/views/configpageView/mapProfiles.coffee

```diff
@@ -0,0 +1,219 @@
+$ = require "jquery"
+_ = require "underscore"
+
+BaseView = require "../baseView.coffee"
+mapProfilesTemplate = require "../../templates/configpageTemplate/mapProfiles.hbs"
+mappingsTemplate = require "../../templates/configpageTemplate/mappings.hbs"
+
+Device = require "../../models/device.coffee"
+MappingProfile = require "../../models/mapping.coffee"
+
+AuthHelper = require "../../helpers/auth.coffee"
+Animations = require "../../helpers/animations.coffee"
+MidiHelper = require "../../helpers/midi.coffee"
+
+module.exports = BaseView.extend
+  el: '#map-profile'
+  template: mapProfilesTemplate
+
+  initialize: (options) ->
+    @parent = options.parent
+
+  render: ->
+    @currentDevice = @parent.getCurrentDevice()
+    if @currentDevice
+      $(@el).html @template({
+        currentDevice: @currentDevice.toJSON()
+      })
+      @initMidiDeviceConnection()
+    else
+      $(@el).html @template({
+      })
+    return @
+  events: ->
+    "shown.bs.collapse .collapse" : "selectProfile"
```

Comment:
Can we make these 2 event triggers more specific? Like add some class since .collapse will be on any collapsible element on that view.